DML/queries/popular_genre.sql

```diff
@@ -0,0 +1,42 @@
+USE library_management_system;
```

Comment:
change all SP files names to match there declared names 
for instance for this SP its file name should be sp_PopularGenreForAGivenMonth
that way its more clear what this file is, i dont have to go into each file to figure out what SP/ FN/ view ... is in that file